shelter/remoteattestation/ratlsclient.c

```diff
@@ -14,8 +14,8 @@
 #define TLSBUF_MAXLENGHT 2048
 
 int ra_tls_echo(int sockfd, enclave_tls_log_level_t log_level,
-		char *attester_type, char *verifier_type, char *tls_type,
-		char *crypto, bool mutual)
+	char *attester_type, char *verifier_type, char *tls_type,
+	char *crypto, bool mutual, unsigned char* sendmsg, unsigned int sendmsglen,  unsigned char* retmsg, unsigned int* recemsglen)
```

Comment:
Fix indent and split into multiple lines.

Reply:
Done